package.json

```diff
@@ -39,15 +38,13 @@
     "grunt-eslint": "^19.0.0",
     "grunt-rollup": "^1.0.1",
     "load-grunt-tasks": "^3.5.2",
-    "rollup-plugin-babel": "^2.7.1",
+    "micromatch": "^2.3.11",
```

Comment:
We don't use this, but I found that it `rollup-plugin-commonjs` was failing because it depended on this indirectly via `rollup-pluginutils` and, for some unknown reason, it was not being installed by npm@5.